shopfloor/services/zone_picking.py

```diff
@@ -1048,19 +1048,23 @@ def set_destination_all(
                     zone_location, picking_type, buffer_lines, message=error
                 )
             # check if the destination location is not the expected one
-            if location != picking_type.default_location_dest_id:
```

Comment:
I saw this check when working on https://github.com/camptocamp/wms/pull/56 which seemed very dubious to me. So I had to check the specs of the zone picking again in https://docs.google.com/presentation/d/1A5TVJXryqod7IwVIl03mDUJIUGzqJ-T2FzSWA90zAvw/edit#slide=id.g6dbc3b0890_0_307, and the implementation follows the specs :shrug: .

Reply:
Then the specs are wrong :p